internal/metric/metric.go

```diff
@@ -67,10 +67,33 @@ func (m MetricsMap) Add(key MetricKey, newVal MetricValue) {
 	m[key] = value
 }
 
+func GetOriginMetadata() Metadata {
+	o := Origin{
+		OriginProduct:       11,
+		OriginSubProduct:    10,
+		OriginProductDetail: 440,
```

Comment:
```suggestion
		OriginProduct:       11,
		OriginSubProduct:    10,
		OriginProductDetail: 440,
```

Can we replace this numbers with some constants? e.g.: GaugeType, DogStatsD, CloudFoundryProduct

Reply:
sure!